reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/mapper/impl/bill/BillEventMapper.java

```diff
@@ -57,11 +57,10 @@ public List<FiscalEvent> transformData(JsonObject data) {
 
             demandJsonArray.forEach(demandJsonElement -> {
                 JsonObject demandJsonObject = demandJsonElement.getAsJsonObject();
-                JsonObject demandAuditDetails = demandJsonObject.get(BILL_DEMAND_AUDIT_DETAILS).getAsJsonObject();
 
                 FiscalEvent fiscalEvent = FiscalEvent.builder().tenantId(applicationConfiguration.getTenantId())
                         .eventType(getEventType())
-                        .eventTime(demandAuditDetails.get(BILL_DEMAND_LAST_MODIFIED_TIME).getAsLong())
+                        .eventTime(Instant.now().toEpochMilli())
```

Comment:
Event time will come from the demand details. Don't set that to current time.
I have commented this in the previous PR. Please resolve all of them.

Reply:
Was mismatched with old one